src/ui/components/App.tsx

```diff
@@ -98,7 +98,7 @@ function App({ children, fontLoading, modal, theme }: AppProps) {
   }, [theme]);
 
   useEffect(() => {
-    if (!isTest() && auth.isAuthenticated) {
+    if (!(isMock() || isTest()) && auth.isAuthenticated) {
```

Comment:
wonder if isMock can be part of isTest... Not sure

Reply:
Yeah, that's probably reasonable. Let me make that change.